src/components/userProfile/components/SideMenu.js

```diff
@@ -17,11 +18,15 @@ export default class SideMenu extends React.Component {
                 mode={INLINE}
             >
                 {
-                    SIDE_MENU_ITEMS.map(sideMenuItem => (
-                        <Menu.Item key={sideMenuItem.label}>
-                            <span><Icon type={sideMenuItem.iconType} />{sideMenuItem.label}</span>
+                    SIDE_MENU_ITEMS.map(({ path, iconType, label }) => (
+                        <Menu.Item key={path}>
+                            <span>
+                                <Icon type={iconType} />
+                                <UserLink key={path} to={path}>{label}</UserLink>
```

Comment:
What are you trying to do here?

Reply:
I intended to use it to access the paths